tern/formats/default/generator.py

```diff
@@ -113,8 +114,14 @@ def get_layer_info_list(layer):
         file_level_licenses = ", ".join(layer_file_licenses_list)
 
     for package in layer.packages:
+        pkg_licenses = package.pkg_license
```

Comment:
To fix the error I mentioned in my [comment](https://github.com/tern-tools/tern/pull/1029/files#issuecomment-958615542), this should be:
```
        pkg_licenses = [package.pkg_license]
```